### ISSUE TITLE
Bell volume support

### ISSUE THREAD

**Issue by scandum** [mintty/mintty#974]
https://invisible-island.net/xterm/ctlseqs/ctlseqs.html
```
CSI Ps SP t
          Set warning-bell volume (DECSWBV), VT520.
            Ps = 0  or 1  ⇒  off.
            Ps = 2 , 3  or 4  ⇒  low.
            Ps = 5 , 6 , 7 , or 8  ⇒  high.

CSI Ps SP u
          Set margin-bell volume (DECSMBV), VT520.
            Ps = 0 , 5 , 6 , 7 , or 8  ⇒  high.
            Ps = 1  ⇒  off.
            Ps = 2 , 3  or 4  ⇒  low.
```

It'd be nice if on / off were supported. Mintty might be able to support volumes by using 6 different sound files?

Additionally \e[?1042h to turn on screen flash on bell, and \e[?1042l to turn off screen flash on bell, do not appear to work.

**Comment by mintty** [comment]
Flash on bell works fine, just tested. Note that the flashing is only applied if the terminal is currently out of focus (same behaviour in xterm).
About the sound, nice idea.

**Comment by BrianInglis** [comment]
Suggest if you go that route, allow just one sound file per sound or bell type (margin, warning, etc.), with a common default. 
Also suggest volume high should be as set by system default (which may be low or muted in office environments; also some of us don't like noisy environments, systems, or blaring audio, especially when ad generated); and volume low should be -3dB, or whatever gives half the output power, of the system default setting. 
It should be possible to mute all sound from each app, as that is the only effective way to deal with sources that do not obey the rules.

**Comment by mintty** [comment]
Unfortunately there is no volume control at the sound output API (PlaySound, MessageBeep, Beep), so different wave files are the only option to implement that feature.
Margin bell is currently not supported.

**Comment by k-takata** [comment]
Audio mixers might be used?
https://docs.microsoft.com/en-us/windows/win32/multimedia/audio-mixers

**Comment by mintty** [comment]
There are also pages like https://docs.microsoft.com/en-us/windows/win32/coreaudio/volume-controls or https://docs.microsoft.com/en-us/windows/win32/coreaudio/endpointvolume-api.
Yet again, instead of simply extending PlaySound(...) with PlaySoundEx(..., volume), MS throughs a huge pile of overengineered and poorly documented API stuff at us. Sample code extends over pages rather than a simple function call. If anyone wants to undergo the nightmare of researching that stuff and coming up with a useful function after days or weeks, I'm open to integrate it...

**Comment by mintty** [comment]
Released 3.1.5.
